packages/SidePanel/src/components/Dialog/Dialog.js

```diff
@@ -80,8 +81,17 @@ function Dialog(props) {
     </React.Fragment>
   );
 
+  React.useEffect(() => {
+    setIsAnimating(true);
```

Comment:
shouldn't this run only when `isOpen === true`? if not this effect will run each time isOpen prop switch no mattering if it's open or close.

Reply:
We want it to animate (run) when it is opening OR closing so I believe it should run for both situations